### PR TITLE
fix: typos and attributions

### PR DESCRIPTION
**Pull request by dannya**

### CODE REVIEW

**Comment by dannya** [comment]
Ready to merge from my side.

File: quotes.json [hunk 1]
```diff
@@ -2549,7 +2541,7 @@
     "text": "If you want to study yourself, look into the hearts of other people. If you want to study other people, look into your own heart."
   },
   {
-    "author": "Fyodor Dostoyevsky",
+    "author": "Fyodor Dostoevsky",
```
Comment (nelsonic) [inline quotes.json:2544]: Both are valid: https://en.wikipedia.org/wiki/Fyodor_Dostoevsky
But happy to use the more popular version. 👌

File: quotes.json [hunk 2]
```diff
@@ -2616,6 +2608,10 @@
     "author": "George Bernard Shaw",
     "text": "The possibilities are numerous once we decide to act and not react."
   },
+  {
+    "author": "George Bernard Shaw",
+    "text": "We do not quit playing because we grow old, we grow old because we quit playing."
```
Comment (nelsonic) [inline quotes.json:2613]: This is one of my favourite quotes of all time. ❤️ 
I want a T-shirt with this on it. 👕 😉 
CC: @iteles / @Stephanymtr any objections to having this on the wall of the "play room" @home? 🏡

**Review by nelsonic — APPROVED** [review]
Nice one @dannya thanks again. 👌

**Pull request merged**